mat64/dense.go

```diff
@@ -65,6 +65,11 @@ type Dense struct {
 	capRows, capCols int
 }
 
+// NewDense creates a new matrix of type *Dense with dimensions r and c. If the mat argument is nil,
+// a new data slice is allocated.
+// Data slice is of Row-major order, i.e., the (i, j) element in matrix should be at (i*c + j)-th
```

Comment:
The data must be arranged in row-major order, i.e. the (i*c + j)-th element in mat is the {i, j}-th element in the matrix.